### DISCUSSION TITLE
what is the simplest way to cache how many users enabled？

### DISCUSSION THREAD

**Discussion by scil** [OrchardCMS/OrchardCore#11263] [Q&A]
I want to display user count on admin dashboard. Is there any simple way to fetch and cache the number? 

I want to use microsoft memorycache, do i need to implement it on my own? is there any methods available in OC?

**Comment by hishamco** [comment]
You consider to use cache TagHelper in Razor View

**Reply by scil** [reply]
Thank you. I serve use count as an API, razor tag useless for me.

And I'd like to know how to update my cahce when a user added or deleted.

**Reply by hishamco** [reply]
> razor tag useless for me.

What about `OrchardCore.DynamicCache`?
